CONTRIBUTING.md

```diff
@@ -345,12 +345,27 @@ The valid captures are:
 
 ### Folds
 
-You can define folds for a given language by adding a `folds.scm` query. This is implemented in Neovim. The only valid capture is
+You can define folds for a given language by adding a `folds.scm` query. This is implemented in Neovim. The only valid capture is `@fold`:
 
 ```query
-@fold ; fold this node
+(function_definition) @fold ; fold this node
```

Comment:
We still scrape this file for CI on Windows, so I think this needs to stay as-is.

Reply:
Yep, which is why we switched to ts_query_ls (but Windows is brittle, period). 😇

Reply:
Oh, wait, I removed that part from the scripts completely since we have a separate lint step now. So you're good!